cashkick-frontend/src/components/atoms/Icon/index.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { ReactNode, ReactElement, ReactPortal } from "react";
+import { SvgIcon } from '@mui/material';
+
+export interface IconProps {
+  iconPath?: string | ReactNode; 
+  cssDesign?: string;  
+};
+
+const Icon: React.FC<IconProps> = ({ iconPath, cssDesign }) => {
+
+  if (React.isValidElement(iconPath)) {
+    return iconPath;
+  }
+
+  if (typeof iconPath === "string") {
+    return (
+      <SvgIcon
+        className={`icon ${cssDesign}`}
+        component="div">
+          <div dangerouslySetInnerHTML={{ __html: iconPath }} />
+      </SvgIcon>
+    );
+  }
+
+  return null;
```

Comment:
please dont return null for any jsx or tsx  component. return an empty react fragment.